src/engine/BytecodeIterator.v3

```diff
@@ -12,7 +12,7 @@
 class BytecodeIterator {
 	def var func: FuncDecl;
 	def var pc: int;
-	private def codeptr = CodePtr.new(null);
+	def codeptr = CodePtr.new(null);
```

Comment:
I think we should add a `nextPC()` method to avoid directly exposing the code pointer here.